Showcase-iOS/CustomView/ActivityIndicatorView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ActivityIndicatorView.swift
+//  Showcase-iOS
+//
+//  Created by Edward Mtshweni on 2018/05/16.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController {
+    
+    func onMainThread(block:@escaping () -> Void) {
+        DispatchQueue.main.async(execute: block)
+    }
+    
+    var customActivityIndicatorView: CustomActivityView? {
+        return CustomActivityView().instatiate()
+    }
+
+    func startAnimating(with title:String) {
```

Comment:
This should be protocol'd out, you just adding random functions to the view controller extensions and not have an api for it.